ANS_JS_template.tex

```diff
@@ -188,116 +188,64 @@ \subsection{Other Research on NRHES Models}
-\item Radiotoxicity of waste streams
-\item Radioactive liquid and gaseous release volumes
-\item Conventional pollutant release over the lifecycle of the nuclear system
-\item Required Enrichment of the fuel
-\item Attractiveness of separated materials for proliferation of nuclear weapons
-\item Difficulties in siting a nuclear power plant, and
-\item Variation in how metrics are calculated.
-\end{itemize}
+In "Modeling the Nuclear Fuel Cycle," Juchau et al. (2010) describe four functions as necessary for a NFCS to meet the desired qualities of an organization proposed at the time called the Simulation Institute for Nuclear Enterprise Modeling and Analysis \cite{Juchau2010}. The four functions, which are reiterated in other synopsis of Nuclear Fuel Cycle Simulators \cite{Wilson2012, NEA2012} include: discrete facilities and material tracking, uncertainty analyses, ability to optimize, and open and accessible with high quality documentation to ensure collaboration. Juchau et al. (2010) reviews the nuclear fuel cycle simulator tools CAFCA, COSI, CEPMNFC, DANESS, NFCSim, VEGAS, VISION, NFCSS, and GENIUS 1. His conclusion is that many of these tools have overlapping functionalities, partially due to the proprietary nature of at least parts of each.
 
-Many of the above listed characteristics are region specific capital and operations and maintenance costs that would also apply to NRHES.  In order to include these characteristics for both the nuclear fuel cycle and NRHES, some measures of uncertainty would need to account for the variability of circumstances. Wilson breaks down the uncertainties into the categories of cost, safety, resource utilization, sustainability, and non-proliferation ~\cite {Wilson2012}. 
+One of the reasons why there are so many nuclear fuel cycle simulators is that many of the countries that currently have NPPs have designed their own simulators. While some NFCSs, such as DANESS, are used in multiple countries, many countries have developed their own NFCSs.  DANESS can be seen in figures 2a and 2b as it has been developed and used both in and out of the US. Figure 2b depicts the non-US NFCSs along with some of their software dependencies. Most of the fuel cycle simulators are not open source or available for download; thus, each nation tends to have proprietary software. As discussed with University of Illinois Professor and developer of the Cyclus nuclear fuel cycle simulator, Dr. Kathryn D. Huff, it is also often more efficient, both in terms of time spent as well as better integration, if the different functions are designed by the same team (\cite{redfoot_huff_2016}). Integrating components from one system into a new system can often be more challenging than building the function from scratch. While sharing code can be challenging amongst various program, some calculations or libraries are commonly used by NFCSs. For example, Figure 3 depicts EVOLCODE, FUTURE, and FAMILY21 as all relying on Origen.
 
```

Comment:
Reviewer #4 Comment 31: Added: While some NFCSs, such as DANESS, are used in multiple countries, many countries have developed their own NFCSs.  DANESS can be seen in figures 2a and 2b as it has been developed and used both in and out of the US.